### PR TITLE
Automatic blog creation from JSON

### PR DESCRIPTION
**Pull request by abac00s**

### CODE REVIEW

**Comment by abac00s** [comment]
Sorry, noticed too late that the blog list has been already reordered. Anyway this should make it a bit easier to add posts...

**Comment by abac00s** [comment]
Some small typos (page1.html instead of page-1.html)

**Comment by abac00s** [comment]
I also separated the blog post template (title, date, contact) from the content of the post. I think it makes easier to add new articles (no need to copy the whole html template and change details manually).

Also it should help with adding markdown interpreter (again, because the file has to contain only the actual content), and comment system (the post_id variable can be used as unique post id, independent from the url)

**Comment by abac00s** [comment]
Fixed small mistake (&hellip; (...) in post descriptions was autoescaped)

File: src/blog-posts.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "excerpt": "Na świecie obecnie żyje 7,5 miliarda ludzi z tego 10,7% (według danych Banku Światowego z 2011) żyje w skrajnym ubóstwie. O skrajnym ubóstwie mówimy wtedy gdy człowiek na przeżycie ma 1,9$ dziennie. Oczywiście możemy tu się spierać, że inaczej żyje się za taką kwotę w Europie niż np. w Azji. Natomiast Bank Światowy uwzględnia różnice w &hellip;",
+        "info": "autor: Maria Gembarzewska-Truong"
+    },
+    "post-1": {
+        "path": "blog/post-1.html",
+        "title": "Myśl jak efektywny altruista",
+        "date": "1.12.2017",
+        "excerpt": "Oto 5 kluczowych pytań, które zadaje efektywny altruizm.",
+        "info": "tłumaczenie: Justyna Ferenc | źródło oryginału: William MacAskill, \"Doing Good Better\""
+    }
```
Comment (marcin-wosinek) [inline src/blog-posts.json]: @Zapsong nowy punkt do dodawania postów:) jak napiszesz coś nowego, będzie potrzeba dodać na górze tej listy. W razie wątpliwości daj znać, i ja dodam

**Comment by marcin-wosinek** [comment]
merged. On the way:
https://travis-ci.org/eapoland/landing-page/builds/356227239
to:
https://eapoland.github.io/landing-page/

File: src/blog-posts.json (hunk 1 above)
Comment (abac00s) [reply src/blog-posts.json]: Oprócz tego trzeba w pliku posta wpisać {% set post_id = "klucz z JSONa" %}

**Pull request merged**